crypto.go

```diff
@@ -158,6 +158,19 @@ func DecryptData(Data map[string]string, ak SymmetricKey) (*map[string]string, e
 	return &decryptedData, nil
 }
 
+// EncryptPrivateKey Encrypts a private key using a keypair.
+func EncryptPrivateKey(privateKey Key, encryptionKeys EncryptionKeys) (string, error) {
+	rawPrivateKeyBytes, err := Base64Decode(privateKey.Material)
```

Comment:
just to clarify... these are abstracted methods that use Base64URL encoding, right?

Reply:
Yes this is used to encrypt the private key for groups

Reply:
Only pushing on this because I think @rosasck might find the source of this answer interesting, but what question do you think Luke asked you / why do you think he asked?